content/projects/katas/level-1/_index.md

```diff
@@ -204,15 +213,15 @@ time
 
 Write a function `combine` that combines two lists by taking alternate elements and returns the result. You can assume the lists are of equal size.
 
-Your combine function should be able to work with lists of unequal size.
+Your combine function should be able to work with lists/arrays of unequal size. Java and JavaScript folks, please use arrays.
```

Comment:
This can be made much more clear. We don't want the learners to have to search for hidden treasures buried in the text in random places. 

It would be better to be bold, clear and consistent.  If you were a UX professional, how would you approach this?

There is currently a section titled "A note to JavaScript developers". Maybe JAvasript-specific stuff should go in there? Maybe there should be other headings for the other languages? 

Eg, something like this:

```
Java specific instructions

- Your function should be able to accept an array of integers as an input
- Your function should return an array of integers

```

Reply:
I placed the java specific instruction inside the same task so learners won't have to look elsewhere for an instruction specific to this task.